### PR TITLE
ELPP-3283 restart server

### PR DESCRIPTION
**Pull request by de-code**

### CODE REVIEW

File: salt/peerscout/init.sls [hunk 1]
```diff
@@ -27,6 +27,7 @@ peerscout-server-systemd-script:
 
 peerscout-server-service:
     service.running:
+        - reload: True
         - name: peerscout-server
```
Comment (giorgiosironi) [inline salt/peerscout/init.sls]: should probably also add `enable: True` so that it is started on boot

**Comment by giorgiosironi** [comment]
The base build from `master` fails also locally, no RDS involved in both cases:
```
[107.23.190.13] out:           ID: peerscout-db-clean
[107.23.190.13] out:     Function: cmd.run
[107.23.190.13] out:         Name: timeout 30 psql --no-password peerscout peerscout -c 'DROP SCHEMA public CASCADE; CREATE SCHEMA public;'
[107.23.190.13] out: 
[107.23.190.13] out:       Result: False
[107.23.190.13] out:      Comment: Command "timeout 30 psql --no-password peerscout peerscout -c 'DROP SCHEMA public CASCADE; CREATE SCHEMA public;'
[107.23.190.13] out:               " run
[107.23.190.13] out:      Started: 16:30:38.124949
[107.23.190.13] out:     Duration: 45.161 ms
[107.23.190.13] out:      Changes:   
[107.23.190.13] out:               ----------
[107.23.190.13] out:               pid:
[107.23.190.13] out:                   22084
[107.23.190.13] out:               retcode:
[107.23.190.13] out:                   1
[107.23.190.13] out:               stderr:
[107.23.190.13] out:                   ERROR:  must be owner of schema public
[107.23.190.13] out:               stdout:
```

**Comment by giorgiosironi** [comment]
Locally, I get
```
peerscout=> \dn
  List of schemas
  Name  |  Owner   
--------+----------
 public | postgres
```
```
peerscout=> \l
                                  List of databases
   Name    |  Owner   | Encoding |   Collate   |    Ctype    |   Access privileges   
-----------+----------+----------+-------------+-------------+-----------------------
 peerscout | foouser  | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 postgres  | postgres | UTF8     | en_US.UTF-8 | en_US.UTF-8 | 
 template0 | postgres | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/postgres          +
           |          |          |             |             | postgres=CTc/postgres
 template1 | postgres | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/postgres          +
           |          |          |             |             | postgres=CTc/postgres
```
so the database is changed ownership to `peerscout`, the app user, but the schema inside it is not (it was usually the easiest thing to drop to catch all tables). I'll go read up on schemas in psql and come back.

@lsh-0 your shiny new `elife.postgresql-appdb` could give us an easy way to clean the db?

**Comment by giorgiosironi** [comment]
A database contains 1 or more schemas, which in turn contains 0 or more tables. Only the schema `public` is used here.
Theory: if we give ownership of the schema to the application user (`pillar.peerscout.db.username`), that would let us execute this step.

**Comment by giorgiosironi** [comment]
`postgres_schema.present` does not change the owner of the schema, just ensures that it exists.

**Comment by giorgiosironi** [comment]
Ok @lsh-0 we definitely need to switch this project to `elife.postgresql-appdb`. Won't fix this problem probably, but better to solve it within the generic code than in this formula only.

**Comment by de-code** [comment]
Closing in favour of https://github.com/elifesciences/peerscout-formula/pull/26

**Pull request closed without merging**